app.rb

```diff
@@ -1,20 +1,107 @@
+require "pry"
+
 require_relative "data"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
 apartments = data[:apartments]
 tenants = data[:tenants]
 
-# First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
-  # Explain how the data is structured
-  # What are the properties for each of the two types of hashes
+require_relative "models/apartment"
+require_relative "models/tenant"
 
+#
+# First, Open the data.rb an inspect the data. Identify and write, in comments, the following:
```

Comment:
Let me know if you'd like to go through any of the features you didn't get to. Happy to walk through a couple.